scripts/shared/cleanup.sh

```diff
@@ -23,7 +21,9 @@ function stop_local_registry {
 
 ### Main ###
 
-run_parallel "{1..3}" delete_cluster
+clusters=($(kind get clusters))
```

Comment:
This is a bit more destructive than what we had, since you might delete kind clusters you didnt plan on deleting.

I guess it should be fine for now but we should probably have a nicer alternative..

Reply:
I guess we'd need to pass the cluster_settings. I think it's reasonable to delete all of them.

Reply:
We can add this is a later PR anyhow since it's not critical